include/dsDisplay.h

```diff
@@ -195,15 +195,39 @@ dsError_t dsDisplayInit();
 
 dsError_t dsGetEDID(intptr_t handle, dsDisplayEDID_t *edid);
 
+/**
+ * @brief Gets the maximum EDID buffer length of connected display device. 
+ * 
+ * This function is used to get the EDID buffer size of the connected display corresponding to
+ * the specified display device handle.
+ *
+ * @param[in] handle       - Handle of the display device
+ * @param[out] maxEDIDSize - length of the EDID buffer data. Min value is 0
+ *
+ * @return dsError_t                        - Status
+ * @retval dsERR_NONE                       - Success
+ * @retval dsERR_INVALID_PARAM              - Parameter passed to this function is invalid
+ * @retval dsERR_NOT_INITIALIZED            - Module is not initialised
+ * @retval dsERR_OPERATION_NOT_SUPPORTED    - The attempted operation is not supported
```

Comment:
why would this occur? `dsERR_OPERATION_NOT_SUPPORTED` ? Was this added to all functions due to AIDL?